src/SarifOutputter.ts

```diff
@@ -0,0 +1,25 @@
+import options from './Options';
+import { convertAxeToSarif } from 'axe-sarif-converter';
+import * as fs from 'fs';
+import * as path from 'path';
+import { promisify } from 'util';
+
+/**
+ * Outputs results from axe to a file in the SARIF format
+ * @param sarifFileName the name of the file that will contain axeResults
+ * @param axeResults results to output
+ */
+export async function exportAxeAsSarifTestResult(sarifFileName: string, axeResults: any) {
```

Comment:
We should be able to import the result type from axe, and use it instead of `axeResults: any`.

```ts
import type { AxeResults } from 'axe-core';
```

and then

```ts
axeResults: AxeResults
```